### PR TITLE
Refactor to simplify OpenXR sample

### PR DESCRIPTION
**Pull request by yl-msft**
Refactor the sample so it's easier to integrate OpenXR sample

## Description

QRCode sample

## Testing
Tested on HL2

## Type of change
- [x] Improvement refactoring

## Supported platforms
HL2

- [ ] All UWP platforms
- [ ] Desktop
- [x] Holographic
- [ ] IoT
- [ ] Xbox
- [ ] 10X

## Supported languages
- [x] C#
- [ ] C++/WinRT

### CODE REVIEW

**Comment by Morgan-6Freedom** [comment]
I really loved to see that merged as soon as possible because I have a HL2 app with QRCode tracking that works with OpenXR and I need this version 🙏

**Review by chgatla-microsoft — APPROVED** [review]

File: Sample/Assets/Scripts/QRCode.cs [hunk 1]
```diff
@@ -4,9 +4,9 @@
 using Microsoft.MixedReality.Toolkit.Input;
 using UnityEngine;
 
-namespace QRTracking
+namespace SampleQRCodes
```
Comment (keveleigh) [inline Sample/Assets/Scripts/QRCode.cs:7]: While we're here, do we want to prepend `Microsoft.MixedReality.` (and to the asmdef below)?

File: Sample/Assets/Scripts/SpatialGraphNode.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                System.Numerics.Matrix4x4.Decompose(newMatrix, out scale, out rotation1, out translation1);
+                translation = new Vector3(translation1.X, translation1.Y, translation1.Z);
+                rotation = new Quaternion(rotation1.X, rotation1.Y, rotation1.Z, rotation1.W);
+                pose = new Pose(translation, rotation);
+                return true;
+            }
+            else
+            {
+                // Debug.Log("Id= " + id + " Unable to locate qrcode" );
+            }
```
Comment (keveleigh) [inline Sample/Assets/Scripts/SpatialGraphNode.cs:74]: Looks like this log was moved into `SpatialGraphNodeTracker`. Should we remove this `else` case?

File: Sample/Assets/Scripts/SpatialGraphNode.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+#endif // WINDOWS_UWP
```
Comment (keveleigh) [inline Sample/Assets/Scripts/SpatialGraphNode.cs:75]:
```suggestion
#endif // WINDOWS_UWP && UNITY_XR_WINDOWSMR
```

**Review by keveleigh — APPROVED** [review]

File: Sample/Assets/Scripts/SpatialGraphNode.cs (hunk 3 above)
Comment (yl-msft) [reply Sample/Assets/Scripts/SpatialGraphNode.cs:75]: Fixed in latest iteration

File: Sample/Assets/Scripts/SpatialGraphNode.cs (hunk 2 above)
Comment (yl-msft) [reply Sample/Assets/Scripts/SpatialGraphNode.cs:74]: Good catch.  Fixed in latest iteration.

File: Sample/Assets/Scripts/QRCode.cs (hunk 1 above)
Comment (yl-msft) [reply Sample/Assets/Scripts/QRCode.cs:7]: Good point.  Fixed in latest iteration.

**Pull request merged**